builtin/aws/ecs/platform.go

```diff
@@ -1340,7 +1340,7 @@ func destroyALB(
 			if err != nil {
 				return err
 			}
-		} else if len(def) > 1 && def[0].ForwardConfig != nil {
+		} else if len(def[0].ForwardConfig.TargetGroups) > 1 && def[0].ForwardConfig != nil {
```

Comment:
```suggestion
		} else if len(def) > 0 && def[0].ForwardConfig != nil && len(def[0].ForwardConfig.TargetGroups) > 1 {
```

Please make this fix. Otherwise, this solution has been tested and works! Thank you!